### PR TITLE
Feature 24 video page

### PR DESCRIPTION
**Pull request by shmato4ek**

### CODE REVIEW

File: frontend/src/styles.scss [hunk 1]
```diff
@@ -1,12 +1,46 @@
...
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
 }
 
 .input-form .mat-form-field-flex {
-    background-color: white;
+  background-color: white;
+}
+
+button {
```
Comment (Jyuart) [inline frontend/src/styles.scss]: Do we really need custom button styles. Material ones are not that good?

File: frontend/src/app/modules/workspace/personal/personal.component.html [hunk 2]
```diff
@@ -1,144 +1,156 @@
+<div>
```
Comment (Jyuart) [inline frontend/src/app/modules/workspace/personal/personal.component.html]: It's pretty difficult to grasp the changes with this diff. Can you describe it in two words?

File: frontend/src/app/modules/workspace/video/video-description/video-description.component.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-video-description',
+  templateUrl: './video-description.component.html',
+  styleUrls: ['./video-description.component.scss'],
+})
+export class VideoDescriptionComponent {
+  @Input() public Username?: string;
+  @Input() public DateOfComment?: string;
```
Comment (Jyuart) [inline frontend/src/app/modules/workspace/video/video-description/video-description.component.ts]:
```suggestion
  @Input() public username?: string;
  @Input() public dateOfComment?: string;
```

?
But maybe I'm wrong. Maybe there is some specific idea behind naming it this way

File: frontend/src/app/modules/workspace/personal/personal.component.html (hunk 2 above)
Comment (shmato4ek) [reply frontend/src/app/modules/workspace/personal/personal.component.html]: I changed the class names of the buttons and moved them to the general styles to be able to style the buttons in the video component in the same way

**Review by Jyuart — DISMISSED** [review]

**Review by Jyuart — DISMISSED** [review]

**Review by Anatolii-Ivanov — DISMISSED** [review]

**Review by Jyuart — DISMISSED** [review]

**Review by Anatolii-Ivanov — DISMISSED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by Jyuart — APPROVED** [review]

**Pull request merged**